indico/modules/events/models/events.py

```diff
@@ -669,6 +669,20 @@ def can_lock(self, user):
         """Check whether the user can lock/unlock the event"""
         return user and (user.is_admin or user == self.creator or self.category.can_manage(user))
 
+    def can_display(self):
+        """Check whether the user can display the event in the category.
+        To be replaced with new permission mechanism: can('display', self, session.user)."""
```

Comment:
i don't think this belongs here; but without this line the docstring is fine

Reply:
Agree, but I saw the "can_lock" that's why I decided to put it there. 
BTW, you want me to remove the "To be replaced with new permission ..." line and keep the rest as is?

Reply:
yes, removing this line is actually all this comment was about :)